### ISSUE TITLE
Website still refers to 1.0.1 documentation

### ISSUE THREAD

**Issue by sharninder** [kairosdb/kairosdb#363]
Where can I see documentation for the new filter aggregation and other newer features?

**Comment by brianhks** [comment]
Documentation is part of the source code in restructured text format.  Here is the docs for the latest master branch: https://github.com/kairosdb/kairosdb/tree/master/src/docs

The docs are pretty readable as plain text.

**Comment by sharninder** [comment]
yes, I know. That's what I referred to. The issue report is just to make sure it gets to a final release in the website. No hurry
